### PR TITLE
filter sanitize before compiling volt

### PR DESCRIPTION
**Pull request by pedro151**

### CODE REVIEW

**Comment by sergeyklay** [comment]
We don't add changes directly to stable branches or `master`. Could you please send this PR to the `3.0.x` branch?

File: phalcon/mvc/view/engine/volt/filter/filterinterface.zep [hunk 1]
```diff
@@ -0,0 +1,9 @@
+namespace Phalcon\Mvc\View\Engine\Volt\Filter;
+
+interface FilterInterface
+{
+	/**
+	 * Filters a value
+	 */
+	public function filter(value) -> string;
+}
```
Comment (Green-Cat) [inline phalcon/mvc/view/engine/volt/filter/filterinterface.zep]: Please end file with newline.

**Comment by pedro151** [comment]
I created a PR in 3.0.x, must have been some mistake, I'm sorry

**Comment by sergeyklay** [comment]
@pedro151 
screenshot at aug 13 00-35-05

**Comment by sergeyklay** [comment]
Could you please send this PR **to the 3.0.x** branch?

**Comment by sergeyklay** [comment]
Closed in favor of #12133

**Pull request closed without merging**